spec/features/flights/index_spec.rb

```diff
@@ -0,0 +1,237 @@
+      end
+
+      within("#flight-#{@flight7.id}") do
+        expect(page).to have_content("Flight Airline: #{@airline4.name}")
+
+        expect(page).to_not have_content(@passenger1.name)
+        expect(page).to_not have_content(@passenger2.name)
+        expect(page).to_not have_content(@passenger3.name)
+        expect(page).to_not have_content(@passenger4.name)
+        expect(page).to_not have_content(@passenger5.name)
+        expect(page).to_not have_content(@passenger6.name)
+        expect(page).to_not have_content(@passenger7.name)
+      end
+
+      within("#flight-#{@flight8.id}") do
+        expect(page).to have_content("Flight Airline: #{@airline4.name}")
+
+        expect(page).to_not have_content(@passenger1.name)
+        expect(page).to_not have_content(@passenger2.name)
+        expect(page).to_not have_content(@passenger3.name)
+        expect(page).to_not have_content(@passenger4.name)
+        expect(page).to_not have_content(@passenger5.name)
+        expect(page).to_not have_content(@passenger6.name)
+        expect(page).to_not have_content(@passenger7.name)
+      end
+    end
+
+    it "has a functional button to remove a passenger from a singular flight" do
+      visit flights_path
+
+      within("#flight-#{@flight3.id}") do
+        expect(page).to have_content(@passenger6.name)
+
+        within("#passenger-#{@passenger6.id}") do
+          expect(page).to have_button("Remove From Flight")
+        end
+      end
+
+      within("#flight-#{@flight5.id}") do
+        expect(page).to have_content(@passenger2.name)
+        expect(page).to have_content(@passenger6.name)
+
+        within("#passenger-#{@passenger2.id}") do
+          expect(page).to have_button("Remove From Flight")
+        end
+
+        within("#passenger-#{@passenger6.id}") do
+          expect(page).to have_button("Remove From Flight")
+        end
+      end
+
+      within("#flight-#{@flight6.id}") do
+        expect(page).to have_content(@passenger6.name)
+
+        within("#passenger-#{@passenger6.id}") do
+          expect(page).to have_button("Remove From Flight")
+        end
+      end
+
+      within("#flight-#{@flight4.id}") do
+        expect(page).to have_content(@passenger4.name)
+        expect(page).to have_content(@passenger5.name)
+        expect(page).to have_content(@passenger6.name)
+
+        within("#passenger-#{@passenger4.id}") do
+          expect(page).to have_button("Remove From Flight")
+        end
+
+        within("#passenger-#{@passenger5.id}") do
+          expect(page).to have_button("Remove From Flight")
+        end
+
+        within("#passenger-#{@passenger6.id}") do
+          click_button "Remove From Flight"
+        end
+
+        expect(page).to have_content(@passenger4.name)
+        expect(page).to have_content(@passenger5.name)
+        expect(page).to_not have_content(@passenger6.name)
+      end
+
+      within("#flight-#{@flight3.id}") do
+        expect(page).to have_content(@passenger6.name)
```

Comment:
Yup, good job testing that we can remove them from one flight, but they are still on the other flight.